README.md

```diff
@@ -33,6 +33,7 @@ Snuba assumes:
 
     mkvirtualenv snuba
     workon snuba
+    make install-python-dependencies
```

Comment:
This will also require `librdkafka` if not installed: https://github.com/getsentry/snuba/blob/1ef1cdf77a173d74d7536b3e75ce3787ad5e096a/Makefile#L36